locale/dbl/LC_MESSAGES/django.po

```diff
@@ -6519,6 +6514,14 @@ msgid "Approvals Since Last Automatic Review"
 msgstr "Ȧƥƥřǿṽȧŀş Şīƞƈḗ Ŀȧşŧ Ȧŭŧǿḿȧŧīƈ Řḗṽīḗẇ"
 
 #: src/olympia/editors/templates/editors/addon_details_box.html
+msgid "Abuse Reports ({num})"
+msgstr "Ȧƀŭşḗ Řḗƥǿřŧş ({ƞŭḿ})"
+
+#: src/olympia/editors/templates/editors/addon_details_box.html
+msgid "Bad User Reviews ({num})"
+msgstr "Ɓȧḓ Ŭşḗř Řḗṽīḗẇş ({ƞŭḿ})"
```

Comment:
You, Sir have damn good eyes. damn it… I kinda expected that tests would fail so imho the site or the string just looks wrong in the end and nothing fails but then again, checking strings kinda is what the debug locale is for. Hmm...